Update AWS deployment documentation (#653)
strapi/documentation: docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/amazon-aws.md

      - Port Range: `8080`
      - Source: `Custom` `0.0.0.0/0, ::/0`
    - Then `Save`

>   - If you have the `ufw` firewall enabled, as recommended by steps below, you will need to configure your settings to include `port 8080`:

Reviewer: ```suggestion
  - If the `ufw` firewall is enabled, configure settings to include `port 8080` by running the following command:
```

To be more compliant with our [12 rules of technical writing](https://handbook.strapi.io/user-success-manual/12-rules-of-technical-writing).
